src/Relude.hs

```diff
@@ -1,35 +1,53 @@
-{-# LANGUAGE Trustworthy #-}
+{-# LANGUAGE Safe #-}
 
 {- |
 Copyright:  (c) 2016 Stephen Diehl
             (c) 2016-2018 Serokell
             (c) 2018-2020 Kowainik
 SPDX-License-Identifier: MIT
-Maintainer: Kowainik <xrom.xkov@gmail.com>
+Maintainer:  Kowainik <xrom.xkov@gmail.com>
+Stability:   Stable
+Portability: Portable
 
 The main module that reexports all functionality. It's considered to be a
 @Prelude@ replacement.
 
-One of the most convenient ways to use @relude@ is via @mixins@ feature. To use
-this feature need to specify @cabal-version: 2.4@ in your package description.
-And then you can add the following lines to the required stanza to replace
+== Usage
+
+=== mixins
+
+One of the most convenient ways to use @relude@ is via @mixins@ feature. This
+feature is available since @Cabal >= 2.2@. In order to use the @mixins@ feature
+one needs to specify supported @cabal-version@ in your package description.
+And then the following lines should be added to the required stanza to replace
 default "Prelude" with @relude@.
 
 @
-mixins: base hiding (Prelude)
-      , relude (Relude as Prelude)
+cabal-version: 3.0
```

Comment:
Maybe relax a bit? Only latest `stack` supports `cabal 3.0` and I guess not a lot of people are using the latest `stack` at the moment...
```suggestion
cabal-version: 2.4
```